README.md

```diff
@@ -0,0 +1,49 @@
+OSIC Reference implementation
+=============================
+
+# What is it?
+
+Reference implementation is opinionated way to deploy OpenStack. One of big
+qualities of OpenStack is it's flexibility, modularity and amount of underlying
+technilogies it can work with. But that also means that it has steep learning
+curve as sheer amount of options it allows is overwhelming (some services can
+have hundreds of them). We want to ease the entry to OpenStack by providing
+what is in our opinion good way to deploy OpenStack. We make sure that it's
+possible to be consumed by majority of users because it will be based on
+opensource as much as it is reasonable. In short, it's OpenStack authors would
+deploy:)
```

Comment:
How are we quantifying this statement "In short, it's OpenStack authors would deploy:)" -- I'm not sure it has a place in this document.